app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/FilesDetailsScreen.kt

```diff
@@ -0,0 +1,109 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.swisstransfer.ui.screen.newtransfer.importfiles
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.infomaniak.multiplatform_swisstransfer.common.interfaces.ui.FileUi
+import com.infomaniak.swisstransfer.ui.components.FileItemList
+import com.infomaniak.swisstransfer.ui.components.SwissTransferTopAppBar
+import com.infomaniak.swisstransfer.ui.components.TopAppBarButton
+import com.infomaniak.swisstransfer.ui.screen.newtransfer.FilesSize
+import com.infomaniak.swisstransfer.ui.theme.Margin
+import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
+import com.infomaniak.swisstransfer.ui.utils.PreviewAllWindows
+
+@Composable
+fun FilesDetailsScreen(
+    filesDetailsViewModel: FilesDetailsViewModel = hiltViewModel<FilesDetailsViewModel>(),
+    fileId: String,
+    navigateToDetails: (String) -> Unit,
+    withSpaceLeft: Boolean,
+    onFileRemoved: ((uuid: String) -> Unit)? = null,
+    onCloseClicked: (() -> Unit),
+    navigateBack: (() -> Unit),
+) {
+    val files by filesDetailsViewModel.getFilesFromUUID(fileId).collectAsStateWithLifecycle(emptyList())
+
+    FilesDetailsScreen(
+        title = {
+            //TODO Get fileId detail to get the title
+            "Some Folder"
+        },
+        files = { files ?: emptyList() },
+        navigateToDetails = navigateToDetails,
+        withSpaceLeft = withSpaceLeft,
+        onFileRemoved = onFileRemoved,
+        onCloseClicked = onCloseClicked,
+        navigateBack = navigateBack,
+    )
+}
+
+@Composable
+private fun FilesDetailsScreen(
+    title: () -> String,
+    files: () -> List<FileUi>,
+    navigateToDetails: (String) -> Unit,
+    withSpaceLeft: Boolean,
+    onFileRemoved: ((uuid: String) -> Unit)? = null,
+    onCloseClicked: (() -> Unit),
+    navigateBack: (() -> Unit),
+) {
+    Column {
+        SwissTransferTopAppBar(
+            title = title(),
+            navigationMenu = TopAppBarButton.backButton { navigateBack() },
+            TopAppBarButton.closeButton { onCloseClicked() },
+        )
+
+        FilesSize(Modifier.padding(vertical = Margin.Medium), files, withSpaceLeft)
```

Comment:
Same here about paddings and `Spacer()`s. Maybe is that one more debatable?

Reply:
I talked about that with @sirambd. We'll keep the modifier but we'll use it inside `FileSize` since the `paddingVertical` is not changing.